transport/conn.go

```diff
@@ -503,6 +504,8 @@ func MakeResponseHandlerWithError(err error) ResponseHandler {
 }
 
 func (c *Conn) sendRequest(req frame.Request, compress, tracing bool) (frame.Response, error) {
+	c.sendController()
+
```

Comment:
How about we add the same **goto control**  flow as in **asyncSendRequest** to avoid returning **errAllStreamsBusy** to user?

Reply:
It's not really possible here, we could be adding it to be on the safe side.
I'm not even sure if it's still possible in the async variant, it may be some sort of a leftover.